workflows/bacterial_genomics/amr_gene_detection/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [0.1] - 05-06-2024
```

Comment:
doesn't match release (1.0) stated in WF file

Reply:
Thanks !